snuba/stateful_consumer/states/bootstrap.py

```diff
@@ -148,8 +148,17 @@ def handle(self, state_data: ConsumerStateData) -> Tuple[ConsumerStateCompletion
         logger.info("Running %r", self.__consumer)
         self.__consumer.run()
 
+        msg = self.__recovery_state.get_active_snapshot_msg()
+        if isinstance(msg, SnapshotLoaded):
+            state_data = ConsumerStateData.snapshot_ready_state(
+                snapshot_id=msg.id,
+                transaction_data=msg.transaction_info,
+            )
+        else:
+            state_data = ConsumerStateData.no_snapshot_state()
```

Comment:
Re: earlier message — seems like this could/should just be `None` but I haven't followed this all the way through it's usages to know if that makes sense.